lib/helper.js

```diff
@@ -38,26 +41,26 @@ var parser = function (pattern, out) {
   return parser(pattern.substring(matches[0].length), out)
 }
 
-var gsParser = function (pattern, out) {
+function gsParser (pattern, out) {
   if (pattern === '**') {
```

Comment:
How about using `const gsParser = (pattern, out) => {` here?

Reply:
(and consistently of course)

Reply:
Fixed. Sorry, for the delay.